server/src/jira/token.guard.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable, ExecutionContext, CanActivate, UnauthorizedException } from "@nestjs/common";
+import { Request } from "express";
+import { SessionData } from "express-session";
+
+@Injectable()
+export class SessionTokenGuard implements CanActivate {
+  async canActivate(context: ExecutionContext) {
+    const req: Request = context.switchToHttp().getRequest();
+    const session: SessionData = req.session;
+    if (!session.user || !session.user.refreshToken || !session.user.accessToken)
+      throw new UnauthorizedException();
```

Comment:
I'm actually super dismayed that our current eslint ruleset allows this. I would always have the `curly` rule on and thought that it was included in the recommended set...

This is justified very well in [eslint/curly docs](https://archive.eslint.org/docs/2.0.0/rules/curly):

> JavaScript allows the omission of curly braces when a block contains only one statement. However, it is considered by many to be best practice to never omit curly braces around blocks, even when they are optional, because it can lead to bugs and reduces code clarity.

But see my next comment first before doing anything about this (: